spec/solver_spec.rb

```
@@ -0,0 +1,29 @@
+require_relative '../src/solver'
+
+describe Solver do
+  solver = Solver.new
+  context 'When testing the factorial method' do
+    it 'should return the factorial of n' do
+      expect(solver.factorial(0)).to eq 1
+      expect(solver.factorial(5)).to eq 120
+      expect(solver.factorial(6)).to eq 720
+      expect(solver.factorial(7)).to eq 5040
```

- [ ] Your factorial test is missing a scenario that tests for a negative number, which should raise an exception. Kindly add it.